### PR TITLE
Fix show template selector when init a project with CLI

### PR DESCRIPTION
**Pull request by andresgutgon**
# What? 
Fix template selector not shown when creating a new project

### TODO
- [x] Fix show template selector when starting a project with CLI
- [x] Show server running URL when showing different version warning

image

image

### CODE REVIEW

File: packages/cli/src/commands/start/index.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ async function welcomeMessage() {
 export default async function startCommand({
   name,
   port,
-  template = TemplateUrl.default,
```
Comment (andresgutgon) [inline packages/cli/src/commands/start/index.ts:37]: Defaults are the evil 😅

File: packages/cli/src/lib/telemetry/index.ts [hunk 2]
```diff
@@ -130,6 +138,20 @@ export class Telemetry {
   private get config(): TelemetryConfig {
     return configStore.get('telemetry')
   }
+
+  // TODO: Provision TELEMETRY_CLIENT_KEY and TELEMETRY_URL
+  // We don't want to use telemetry in development mode
+  // we will provision this for the published CLI
+  private get credentials(): TelemetryCredentials | undefined {
+    const clientKey = process.env.TELEMETRY_CLIENT_KEY ?? CLIENT_KEY
+    const clientUrl = process.env.TELEMETRY_URL ?? ANALYTICS_URL
```
Comment (andresgutgon) [inline packages/cli/src/lib/telemetry/index.ts:147]: @geclos we are using Telemetry even when we develop in the monorepo. We need to provision these two env variables in GitHub to only enable telemetry when we release the CLI package. I'll do something similar with Sentry

File: packages/cli/tests/e2e/start.cjs [hunk 3]
```diff
@@ -4,7 +4,15 @@ const { spawn, spawnSync } = require('child_process')
...
-spawnSync('node', [path.join(__dirname, '../../dist/cli.js'), 'telemetry', '--disable'])
+// Disable telemetry
+spawnSync(
+  'node',
+  [
+    path.join(__dirname, '../../dist/cli.js'),
+    'telemetry',
+    '--disable'
+  ]
+)
```
Comment (andresgutgon) [inline packages/cli/tests/e2e/start.cjs:15]: Telemetry now has a good test coverage. Also with the env variables this won't be necessary. 

## When telemetry should be used
1. When developing the monorepo 🚫 
2. When running CI test or linter 🚫 
3. When publishing the CLI package in NPM ✅

**Review by csansoon — DISMISSED** [review]

File: packages/cli/src/lib/telemetry/index.ts (hunk 2 above)
Comment (geclos) [reply packages/cli/src/lib/telemetry/index.ts:147]: create an issue please 😅 otherwise i won't remember

Comment (andresgutgon) [reply packages/cli/src/lib/telemetry/index.ts:147]: [Done](https://github.com/latitude-dev/latitude/issues/207)

**Pull request merged**